app/services/ccr/daily_attendance_adapter.rb

```diff
@@ -24,23 +38,28 @@ def attendances_for(claim)
 
     def attendances
       if daily_attendance_uplifts?
-        daily_attendance_uplifts + DAILY_ATTENDANCES_IN_BASIC
+        daily_attendance_uplifts + daily_attendances_in_basic
       else
-        [trial_length, DAILY_ATTENDANCES_IN_BASIC].compact.min
+        [trial_length, daily_attendances_in_basic].compact.min
       end
     end
 
     private
 
-    # The first 2 daily attendances are included in the Basic Fee (BABAF)
-    DAILY_ATTENDANCES_IN_BASIC = 2
+    def daily_attendances_in_basic
+      claim.fee_scheme.eql?('fee_reform') ? 1 : 2
```

Comment:
Just FYI, going forward I'm thinking about deprecating the use of `fee_scheme.eql?('fee_reform')` and moving towards other approach of determining the fee scheme type.

Reply:
yeah, was not keen on using this myself. `claim.fee_scheme?(10)` or some such would be much clearer.